src/App.tsx

```diff
@@ -1,7 +1,33 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+export class App extends React.Component {
+  state = {
+    pressedKey: '',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.recordKey);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.recordKey);
+  }
+
+  recordKey = (event: KeyboardEvent) => {
+    this.setState({ pressedKey: event.key });
+  };
+
+  render() {
+    return (
+      <div className="App">
+        {this.state.pressedKey
+          ? (
+            <p className="App__message">{`The last pressed key is [${this.state.pressedKey}]`}</p>
+          )
+          : (
+            <p className="App__message">Nothing was pressed yet</p>
+          )}
+      </div>
```

Comment:
you can use destructuring, it's optional but I think it's better
   const { pressedKey } = this.state;
```suggestion
  render() {
    return (
      <div className="App">
        {pressedKey
          ? (
            <p className="App__message">{`The last pressed key is [${pressedKey}]`}</p>
          )
          : (
            <p className="App__message">Nothing was pressed yet</p>
          )}
      </div>
```